### ISSUE TITLE
[WARRIOR][BUG][DESARMED]

### ISSUE THREAD

**Issue by warriorwoltk** [SunwellTracker/issues#2602]
1 Actually when you get desarmed by warrior you can't shield block or shield wall wich is wrong because according to tooltip it only remove main hand weapon

I think i can't reflect too while desarmed by a warrior since it affect my shield too

2 When you get desarmed by shadow priest with psychic horror you can't shield block or shield wall wich is wrong because according to tooltip target only drop their main hand weapon

I think i can't reflect too while desarmed by a priest since it affect my shield too

3 Didn't experienced it at the moment but i guess it's same deal when you get desarmed by hunter spell and i probably can't reflect too

4 When you get desarmed bu rogue you can't shield block and probably can't reflect too

**Comment by Aridam** [comment]
You're misunderstanding what's said in the report and in the links provided there.

The way Disarm worked was it used to only remove your currently-equipped Main Hand and Ranged Slot Weapons -- Swifty was one of the people who found and popularized a bug with it that allowed you to bypass it completely, 

Eventually that got ninja-fixed, and as seen in some of the links, people have reported being unable to switch to a Shield if they got Disarmed while wielding a 2H (unless Fury), because they need to swap out their 2H so they can use their Shield, which they can't because the Disarm is NOT ALLOWING IT, hence they get locked out.

This is neither a fault by Blizz nor an invalid bugreport by me -- it was how Disarm's mechanic got changed. It _still_ removed your MH and Ranged Weapons, but it did so by directly restricting access to those two Item Slots.

If you don't understand, go and re-read again, as well as the info in the links, and if need be, use a translator or something. Just don't go spreading misinformation and wasting people's time.

**Comment by warriorwoltk** [comment]
It's not restricting the way to swap weapons while desarmed, provide something more than just average players talking about game mechanics they don't know, that forum was full of average players asking for tips etc 

I can provide video of players actually swapping weapons while desarmed on every competitive 3.3.5a servers, and i assume if it was a bug or an abuse everyone would have report it since everyone complain about arms warrior on pvp scene

Also i readed all patch notes for disarm, dismantle and general change and nothing appear to change it

**Comment by Aridam** [comment]
Buddy, you're pullin' some bad strings here.

It _is_ doing that. After a patch in WotLK, many players (like all the ones on RETAIL-FRIGGIN'-ERA WOTLK WOW FORUMS) have reported this difference and confirmed it with one-another. There was more to removing the bug Swifty showcased, and I found that out myself in Dragonblight because that's the first time I'd seen this mechanic - - thought it was a bug, turned out it wasn't.

This was not listed in any patchnotes. If stuff like this and more was listed as such, don't you think all of these issues would be so much easier to find, diagnose and fix? Blizzard themselves didn't list them. That's why they're called "ninja-fixes".

Other private servers cannot be used as examples, "competitive" or not. Most you could use is correlate bugreports of it in other servers for their popularity, provided evidence there which could possibly be used hee, and other diagnosable stuff.

Yeah "working" as intended like how someone made a false report on Heroic Throw resetting the Swing Timer was a bug, misleading the devs into "fixing" it, JK turns out it was working as intended all along.

Also how do you know if those players were average or not? You sound pretty friggin' average to me, especially with your "DESARM" spelling. Unless _that's_ a bug too?

Also, for 5k+ players currently here, the bugtracker is pretty deserted in terms of activity, so you'd think everyone would report it, but no. In fact there've been numerous invalid reports. Like this one. Go figure. 

Now could you get your head out of your own ass.

**Comment by warriorwoltk** [comment]
I gonna provide some real proofs instead of players saying " i can't do it anymore " on some sub forum, if devs can make heroic throw not reseting swing because someone report it as bugged your own report don't have any value for me

> Most you could use is correlate bugreports of it in other servers for their popularity

Disarmed mechanic work like i said on thoses servers for years, probably more than 6y, it's also competitive pvp servers wich mean if it wasn't working as intented then they would have report it since it directly affect one of most played class in 2s, imagine launching 10k dollar prize pool, with overpowered bug affecting one class present in every team

So if i spell wrongly some ingame spell since it's not my native language im average player ? That 50 iq logic

**Comment by Aridam** [comment]
Yeah sure go ahead with your delusional self. Just don't go misleading the deva into turning the server into a royal unplayable mess just because you disagree with one simple mechanic you dislike because you're too casual.

**Comment by warriorwoltk** [comment]
> "If _**I remember correctly**_ pre-3.2 you could still equip a shield if your 2h was disarmed by dismantle/psychic Horror. Good players using this well, timing dismantles with shatters etc.. really annoying :S"

> "as titan grip, **_you could sometimes take out a shield_**. and that was the only exception"
"pre 3.2 you could trinket psychic horror if it disarmed your 2h and spell reflect. Now it just says "can't do that while disarmed"."

they'r clearly not sure about it, this " sometimes " you could do it end all form of values

seems like in that specific thread they talk only about dismantle rogue's spell

> you can if you swap like as soon as you get dismantled and you swap to 1h+shield, you can keep your shield sometimes

again ! sometimes you could still keep shield sometimes

in same thread there's players who don't know if you keep shield wall buff after dismantle, if you get armor from shield, etc and you take this as granted ...

just get away with you shitty trash talk / behavior keep it when i slay you in all brackets

**Comment by Aridam** [comment]
Close this -- not a valid issue as seen in #2613.
